APIBacklog/documentation/APIBacklog.md

```diff
@@ -3,51 +3,61 @@
+| Simple Edge Discovery | Vodafone | N/A | N/A  | 5GFF, Vodafone | Approved (N/A)  | [EdgeCloud/Discovery](https://github.com/camaraproject/EdgeCloud/tree/main/code/API_definitions/Discovery) | simple-edge-discovery | [Link](https://github.com/camaraproject/EdgeCloud/tree/main/documentation) | v0.9.1 | [YAML](https://github.com/camaraproject/EdgeCloud/blob/main/code/API_definitions/Discovery/simple_edge_discovery.yaml) | N/A | N/A | N/A| N/A|<!---MEC Management --->
+| MEC Management | Telefonica | N/A | N/A  | GSMA-OGW, Telefonica | Approved (N/A) |[EdgeCloud/Discovery](https://github.com/camaraproject/EdgeCloud/tree/main/code/API_definitions/Discovery)  | mec-exposure-and-experience-management | [Link](https://github.com/camaraproject/EdgeCloud/tree/main/code/API_definitions/Discovery) | v0.8.1 | [YAML](https://github.com/camaraproject/EdgeCloud/blob/main/code/API_definitions/Discovery/MEC%20exposure%20and%20experience%20management.yaml) | N/A | N/A | N/A| N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Device Location-->%%%%%%%%%%%%%%%%%%%%%
+| Device Location | DTAG | 2022/01/11 | N/A | DTAG, Orange, Telefonica, Vodafone | Approved (2022/04/17) | [DeviceLocation](https://github.com/camaraproject/DeviceLocation) | location-verification | [Link](https://github.com/camaraproject/DeviceLocation/blob/main/documentation) | v0.2.0-wip | [YAML](https://github.com/camaraproject/DeviceLocation/blob/main/code/API_definitions/location-verification.yaml) | N/A | N/A | N/A|
+| |  |  |  |  |   | |  |  | v0.1.0 | [YAML](https://github.com/camaraproject/DeviceLocation/blob/release-v0.1.0/code/API_definitions/location.yaml) | N/A | N/A | N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Device Status-->%%%%%%%%%%%%%%%%%%%%%
+| Device Status | DTAG | 2022/01/11 | N/A | DTAG, Orange, Telefonica, Vodafone | Approved (2022/04/17) |  [DeviceStatus](https://github.com/camaraproject/DeviceStatus) | device-status | [Link](https://github.com/camaraproject/DeviceStatus/tree/main/documentation) | v0.5.0-wip | [YAML](https://github.com/camaraproject/DeviceStatus/blob/main/code/API_definitions/device-status.yaml) |  N/A |  N/A |  N/A|
+| |  |  |  |  |  |   |  |  | v0.4.1 |  N/A | N/A | N/A | N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Home Devices Quality on Demand --->
+| Home Devices Quality on Demand | Telefonica | 2022/09/14 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_HomeDevicesQoD_Telefonica.md) | DTAG, KDDI, Telefonica |  Approved (2022/10/13) | [HomeDevicesQoD](https://github.com/camaraproject/HomeDevicesQoD) | home_devices_qod | [Link](https://github.com/camaraproject/HomeDevicesQoD/blob/main/documentation/API_documentation) | v0.2.0 | [YAML](https://github.com/camaraproject/HomeDevicesQoD/blob/main/code/API_definitions/home_devices_qod.yaml) |  N/A |  N/A |  N/A|
```

Comment:
Links should point to the release branch of v0.2.0, not to main. The API has inline documentation, link from the release notes: https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/HomeDevicesQoD/release-0.2.0/code/API_definitions/home_devices_qod.yaml